pkg/storaged/lvm2/volume-group.jsx

```diff
@@ -67,6 +68,7 @@ function vgroup_rename(client, vgroup, card) {
         ],
         Action: {
             Title: _("Rename"),
+            Danger: contains_rootfs(client, vgroup.path) ? _("This volume group contains the root filesystem. Renaming it might require further changes to the bootloader configuration or kernel command line.") : null,
```

Comment:
I realize we can't split the translatable string, but it could at least go into the next line to make the line a litte less long.

Reply:
Yes, done.